inst/stan/functions/expected_obs.stan

```diff
@@ -58,37 +62,38 @@
  * rep_lh <- rep(0, 30)
  * rep_lh[c(6, 12, 16)] = 2
  * rep_lh[c(2, 20)] = -2
- * round(exp(expected_obs(tar_obs, equal_lh + rep_lh, 0)), 3)
+ * round(exp(expected_obs(tar_obs, equal_lh + rep_lh, 0, 0, [0])), 3)
  * # 0.033 0.005 0.034 0.034 0.034 0.202 0.027 0.027 0.027 0.027 0.027 0.151
  * # 0.021 0.021 0.021 0.106 0.014 0.014 0.014 0.002 0.016 0.016 0.016 0.016
  * # 0.016 0.016 0.016 0.016 0.016 0.016
  *
  * Example combining both date of reference and date of report effects
- * eobs <- exp(expected_obs(tar_obs, ref_lh + rep_lh, 0))
+ * eobs <- exp(expected_obs(tar_obs, ref_lh + rep_lh, 0, 0, [0]))
```

Comment:
Can you add an example here showing what this does (i.e below this example) when it is on